# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../live/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- live/liveMedia/include/AVIFileSink.hh_orig	2006-03-19 17:05:14.748308500 +0100
+++ live/liveMedia/include/AVIFileSink.hh	2006-03-19 17:05:48.142395500 +0100
@@ -79,7 +79,7 @@
 
   unsigned addWord(unsigned word); // outputs "word" in little-endian order
   unsigned addHalfWord(unsigned short halfWord);
-  unsigned AVIFileSink::addByte(unsigned char byte) {
+  unsigned addByte(unsigned char byte) {
     putc(byte, fOutFid);
     return 1;
   }
--- live/liveMedia/include/QuickTimeFileSink.hh_orig	2006-03-19 17:03:48.146896250 +0100
+++ live/liveMedia/include/QuickTimeFileSink.hh	2006-03-19 17:04:44.786436000 +0100
@@ -87,7 +87,7 @@
 
   unsigned addWord(unsigned word);
   unsigned addHalfWord(unsigned short halfWord);
-  unsigned QuickTimeFileSink::addByte(unsigned char byte) {
+  unsigned addByte(unsigned char byte) {
     putc(byte, fOutFid);
     return 1;
   }
